app/assets/javascripts/member-facing/backbone/survey.js

```diff
@@ -0,0 +1,167 @@
+let ErrorDisplay = require('shared/show_errors');
+let MobileCheck = require('member-facing/backbone/mobile_check');
+const GlobalEvents = require('shared/global_events');
+
+const Survey = Backbone.View.extend({
+
+  el: '.survey',
+  HIDDEN_FIELDS: ['source', 'referrer_id'],
+
+  events: {
+    'click .survey__skip-button': 'skipSection',
+    'ajax:success form.survey__form': 'handleSuccess',
+    'ajax:error form.survey__form': 'handleFailure',
+    'ajax:send form.survey__form': 'handleSend',
+  },
+
+  // options: object with any of the following keys
+  //    source: the referring source to save
+  //    referrer_id: the champaign id of the referrer
+  //    prefill: an object with fields that will prefill the form
+  //    followUpUrl: the url to redirect to after the survey is completed
+  initialize(options={}) {
+    this.$forms = this.$('.survey__form');
+    if (!MobileCheck.isMobile()) {
+      this.selectizeCountry();
+    }
+    this.insertHiddenFields(options);
+    this.prefill(options.prefill);
+    this.revealFirstForm();
+    this.submitFirstFormIfComplete();
+    this.followUpUrl = options.followUpUrl;
+    GlobalEvents.bindEvents(this);
+  },
+
+  revealFirstForm() {
+    this.$('.survey__form').first().removeClass('hidden-closed');
```

Comment:
it's actually a [little less efficient](https://api.jquery.com/first-selector/) -
> Because :first is a jQuery extension and not part of the CSS specification, queries using :first cannot take advantage of the performance boost provided by the native DOM querySelectorAll() method. To achieve the best performance when using :first to select elements, first select the elements using a pure CSS selector, then use .filter(":first").